frontend/lib/api/form/pollingstation/PollingStationFormController.tsx

```diff
@@ -297,6 +230,11 @@ export function PollingStationFormController({
     [currentForm, formState],
   );
 
+  if (values === undefined) {
+    // loading
+    return null;
```

Comment:
In #263 we decided that we want a white (empty) page while loading, nothing else.

Reply:
yes, assumption is that loading of pages will be very fast because everything from local network and not loads of data. White page is a bit cleaner / less jumpy then showing something that you'll never be able to read